src/API/Plugin.php

```diff
@@ -65,13 +68,20 @@ public function createAPISuccessResponse($result)
         );
     }
 
-    public static function createPluginSettingObject($pluginSettingKey, $value, $editable, $modified_on)
+    /**
+     * @param $pluginSettingKey
+     * @param $value
+     * @param $editable
+     * @param $modified_on
+     * @return array
+     */
+    public function createPluginSettingObject($pluginSettingKey, $value, $editable, $modified_on)
```

Comment:
I know removing `static` here will break if you upgrade WordPress again.  Hopefully I'll have reworked that code by then to be class based but if not it will force us to rework it.